### PR TITLE
[Term Entry] learning-rate-schedule.md (created)

### LINKED ISSUE #4530: [Term Entry] Neural Networks ai Learning Rate Scheduling
### Reason/inspiration (optional)

"We would like a new term entry in the `AI` concept for [neural-network](https://www.codecademy.com/resources/docs/ai/neural-networks): Learning Rate Schedule. The entry should go in a new file under `docs/content/ai/concepts/neural-networks/terms/learning-rate-schedule/learning-rate-schedule.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use concept for stochastic gradient descent.
- Should contain information for this concept and syntax.
- Should have a Python codebyte with a working code snippet showcasing this concept.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by FuzzyNum**
### Description

Created file learning-rate-schedule.md in docs/content/ai/concepts/neural_networks/terms/learning-rate-scheduling

### Issue Solved
Closes #4530  

### Type of Change
- Adding a new entry
- Updating the documentation

### Checklist

- [✅] All writings are my own.
- [✅] My entry follows the Codecademy Docs style guide.
- [✅] My changes generate no new warnings.
- [✅] I have performed a self-review of my own writing and code.
- [✅] I have checked my entry and corrected any misspellings.
- [✅] I have made corresponding changes to the documentation if needed.
- [✅] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✅] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✅] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by raditya1117 — CHANGES_REQUESTED** [review]
Hi @FuzzyNum,
Thanks for your contribution. The content looks good. However, I have proposed some minor changes. Please go through the comments and make the suggested changes as required.

**Comment by FuzzyNum** [comment]
Thank you for your edits. I found them helpful and clarifying. I chose to apply all of them, let me know if there's any more changes you deem necessary!

**Comment by raditya1117** [comment]
@FuzzyNum, There are still some unresolved comments. Could you please check them?

**Comment by raditya1117** [comment]
@ishg-153 
The issue mentioned for this PR was for a term entry, and the actual PR was for a Topic entry. As you rightly corrected, the PR should have been for a Term Entry. However, the content for the given topic cannot be written as a term entry due to its length, and I had considered it a topic entry as per the PR, which was a miss. Can we accept it as a Topic Entry instead of a Term entry? If we must convert it to a Term Entry, we must modify the format and remove most of the content. @rashi95

**Comment by ishg-153** [comment]
Hey @raditya1117! I see what you mean. But since this term entry belongs under neural networks, we can't just make it a topic entry. A topic entry can only be made when that topic has many concepts within it and further, those concepts have a lot of terms that can have their individual entries.

**Comment by FuzzyNum** [comment]
Hey @ishg-153, I intended to submit it as a term entry. What changes should be made to qualify it as such? In the content standards guide, I don't see a clear explanation of the conflict. I'm happy to update its format or modify it if necessary.

**Comment by ishg-153** [comment]
@raditya1117

**Comment by Radhika-okhade** [comment]
Closing it because the PR was for a term entry, and the actual PR was for a Topic entry.
